### PR TITLE
NAS-102494 / 11.3 / Allow lzma compression for importing/exporting jails

### PR DESCRIPTION
**Pull request by sonicaj**
This PR aims to expose option to allow lzma compression for import/export of jails.

### CODE REVIEW

File: iocage_lib/iocage.py [hunk 1]
```diff
@@ -1386,9 +1388,11 @@ def get(
 
             return jail_list
 
-    def import_(self):
+    def import_(self, compression_algo='zip'):
```
Comment (william-gr) [inline iocage_lib/iocage.py]: Is there a way to autodetect the file instead?

Comment (sonicaj) [reply iocage_lib/iocage.py]: Well we can but that means an exception if we have 2 exported jails with similar names but different algorithms. I guess i raise an exception in that case and that will allow this to automatically proceed if there are no conflicts. Thoughts ?

**Review by william-gr — COMMENTED** [review]
I find it weird that we cannot provide a path of the image we want to import?

**Comment by sonicaj** [comment]
Well we have a defined dataset for it right now where all images must live. But for import cases it indeed is. Would you like me to add an option ?

**Comment by william-gr** [comment]
Yes, also seems easier for use case instead of having to move it first to images dataset.

**Review by william-gr — APPROVED** [review]

**Pull request merged**